hovercraft/local_directives/__init__.py

```diff
@@ -0,0 +1,40 @@
+import os
+import tempfile
+from collections import defaultdict
+
+# import all directives defined in local_directives directory
+d = os.path.dirname(os.path.realpath(__file__)) # local_directives directory
+__all__ = [] # list of local directive modules
+for item in os.listdir(d):
+    if os.path.isdir(os.path.join(d, item)) and item != 'sandbox':
+        __all__.append(item)
+from . import *
```

Comment:
No star imports.

Reply:
Okay. The purpose is to allow people to be able to just put a folder containing a new directive into the local_directives folder and have it just work. This of course assumes that there is a file inside that new folder called __init__.py. Is there a different naming scheme that would accomplish the same thing without star imports or __init__'s. 

I have tried "import ." instead of "from . import *", but it doesn't work.

Reply:
Plugins need to have either a definable directory to be located in (or a standardized Python name space for Hovercraft to discover it). The plugin system must know what plugins are there and what their names are.

I'm also not convinced a plugin system is necessary at this point. I don't really see the need to add arbitrary directives. But if more people want it, I'm OK with it.